app/routes/upload.step1.tsx

```diff
@@ -0,0 +1,130 @@
+import { ActionFunctionArgs } from '@remix-run/node'
+import { Form, redirect, useNavigation, useRouteError } from '@remix-run/react'
+import { useState } from 'react'
+import { v4 as uuidv4 } from 'uuid'
+import { Button } from '~/components/ui/button'
+import { cn } from '~/lib/utils'
+import { supabase } from '~/supabaseClient'
+
+// Action to handle form submission
+export const action = async ({ request }: ActionFunctionArgs) => {
+  const body = await request.formData()
+  const file = body.get('cv-file') as File | null
```

Comment:
**issue**: The error message 'Could not get file.' is not user-friendly. Consider providing a more descriptive error message.